src/ui/settings-dialog.cpp

```diff
@@ -148,17 +146,35 @@ void SettingsDialog::updateSettings()
         msg = tr("language");
     }
 
+    if(language_changed && gui->yesOrNoBox
+            (tr("You have changed language, Restart to apply it?"), this, true)) {
+        gui->restartApp();
+    }
+
+
+#if defined(_MSC_VER)
+
+    bool seadrive_root_changed = false;
+    if (mShowCacheDir->text() != current_seadrive_root_ ) {
+        seadrive_root_changed = true;
+        mgr->setSeadriveRoot(mShowCacheDir->text());
+    }
+
+    if (seadrive_root_changed && gui->yesOrNoBox(tr("You have changed seadrive root. Restart to apply it?"), this, true)) {
```

Comment:
You have changed SeaDrive cache folder. Restart to apply it?

Reply:
ok